include/display/README.md

```
@@ -0,0 +1,71 @@
+5. In your *main.c*: #include "lvgl/lvgl.h"
+6. In your *main function*:
+   * lvgl_init();
+   * tick, display and input device initialization (see above)
+7. To **test** create a label: `lv_obj_t * label = lv_label_create(lv_scr_act(), NULL);`
+8. In the main *while(1)* call `lv_task_handler();` and make a few milliseconds delay (e.g. `my_delay_ms(5);`)
+9. Compile the code and load it to your embedded hardware
+
+## PC Simulator
+If you don't have got an embedded hardware you can test the graphics library in a PC simulator. The simulator uses [SDL2](https://www.libsdl.org/) to emulate a display on your monitor and a touch pad with your mouse.
+
+There is a pre-configured PC project for **Eclipse CDT** in this repository: https://github.com/littlevgl/pc_simulator
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)